cmd/helpers.go

```diff
@@ -143,6 +143,7 @@ func BuildDocker(binaryName string, buildMode string, projectOptions *ProjectOpt
 		"-v", fmt.Sprintf("%s:/build", filepath.Join(fs.Cwd(), "build")),
 		"-v", fmt.Sprintf("%s:/source", fs.Cwd()),
 		"-e", fmt.Sprintf("LOCAL_USER_ID=%v", userid),
+		"-e", fmt.Sprintf("FLAG_TAGS=\"%s\"", projectOptions.Tags),
```

Comment:
I don't think the escaped quotes are needed here from what I can see https://github.com/karalabe/xgo/blob/c5ccff8648a77686d274c1984e1cd6319736f5cb/xgo.go#L297

Reply:
I have fixed it :)